packages/angular/angular.json

```diff
@@ -269,6 +269,118 @@
           }
         }
       }
+    },
+    "clarity-adoption": {
+      "projectType": "application",
+      "schematics": {
+        "@schematics/angular:component": {
+          "style": "scss"
+        }
+      },
+      "root": "projects/clarity-adoption",
+      "sourceRoot": "projects/clarity-adoption/src",
+      "prefix": "app",
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:browser",
+          "options": {
+            "outputPath": "dist/clarity-adoption",
```

Comment:
This should be `"outputPath": "../../dist/website/adoption-guide",` so it outputs into the website project.